### DISCUSSION TITLE
Material You colors?

### DISCUSSION THREAD

**Discussion by JonathanBout** [yiszza/Material.Components.Maui#50] [Q&A] [answered]
Does this support material you colors? If yes, how can I implement that?
(Material You is the OS color scheme which in Android 12+ can be set by the user.)

**Comment by ziomek64** [comment, marked as answer]
Make at least one search beforehand: https://github.com/mdc-maui/mdc-maui/issues/36
So it's possible with someone's library, haven't tried though

**Reply by JonathanBout** [reply]
I searched for it but I didn't come across that one 🤷‍♂️ Thank you anyways
